scripts/bootstrap

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+yarn install
```

Comment:
@neilbmclaughlin added the submodule initialisation into profiles-db(https://github.com/nhsuk/profiles-db/pull/55), would be good to add in here too?